app/domain/mailing_lists/subscribers.rb

```diff
@@ -32,6 +32,72 @@ def people_as_configured
     filter_chain.filter(@people_scope.where(id: Person.find_by_sql(people_cte.to_sql)))
   end
 
+  def person_subscribers(condition)
+    condition.or("subscriptions.subscriber_type = ? AND subscriptions.excluded = ? " \
+      "AND subscriptions.subscriber_id = people.id ", Person.sti_name, false)
+  end
+
+  def group_subscribers(condition)
+    sql = <<~SQL.split.join(" ")
```

Comment:
```suggestion
    sql = <<~SQL.squish
```

https://devdocs.io/rails~7.1/string#method-i-squish löst das allgemeiner.